internal/plugintypes/timestamp.go

```diff
@@ -12,13 +12,29 @@ type TimestampedBig struct {
 	Value     cciptypes.BigInt `json:"value"`
 }
 
+// TimestampedUnixBig Maps to on-chain struct
+// https://github.com/smartcontractkit/chainlink/blob/37f3132362ec90b0b1c12fb1b69b9c16c46b399d/contracts/src/v0.8/ccip/libraries/Internal.sol#L43-L47
+//
+//nolint:lll //url
+type TimestampedUnixBig struct {
```

Comment:
Can we just use this in the observations instead of having two different types?

Reply:
In a next PR. For now let's make this PR more about fixes. Changing this will change a lot of things about observations with timestamps